gap/generic/kernel.gi

```diff
@@ -77,36 +77,35 @@ end );
 # Computes randomly (it might underestimate) the normal closure of <list>
 # under conjugation by the group generated by <grpgens>.
```

Comment:
Maybe add some information about <n> from the input?

Reply:
Jepp, good point.

Reply:
I second that. It would be nice to know what the error probability is in terms of `n` (and the maximal length of subgroup chains in `<grpgens>`).